### DISCUSSION TITLE
Benchmarking on RK3399

### DISCUSSION THREAD

**Discussion by heiher** [heiher/hev-socks5-tunnel#20] [Show and tell]
Tun2socks benchmark results on NanoPi M4(AArch64 RK3399):

* hev-socks5-tunnel
* xjasonlyu-tun2socks
* go-tun2socks
* leaf-tun2socks
* badvpn-tun2socks

### Speed

![rk3399-speed](https://user-images.githubusercontent.com/1407733/221888472-9c4c790f-b193-4926-814c-a87a06b703bf.png)

### CPU Usage

![rk3399-cpu](https://user-images.githubusercontent.com/1407733/221888507-512255a2-ed3c-457a-9762-821ee0ba93bd.png)

**Comment by CCCAUCHY** [comment]
能否贴出iperf3的零拷贝回环性能? 我认为用性能损失率来评价比较直观

**Reply by heiher** [reply]
后面可以加到README那个维度的评测中。当前这个是评测在低性能的设备上经过1Gbps物理网口的性能损失。
